cli_tools_tests/module/import/file_inflation_test.go

```diff
@@ -0,0 +1,129 @@
+//  Copyright 2020 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package import_test
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/google/uuid"
+
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/common/imagefile"
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/common/utils/logging"
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/common/utils/storage"
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/gce_vm_image_import/importer"
+	daisycompute "github.com/GoogleCloudPlatform/compute-image-tools/daisy/compute"
+)
+
+const (
+	workflowDir = "../../../daisy_workflows"
+)
+
+func TestDaisyFileInflation(t *testing.T) {
```

Comment:
For module tests, due to the possibility of modules being defined loosely, it's not always clear what they tests without reading through test code. Please add docs with big picture scope of the test (and for future tests)

Reply:
Good feedback!
 - Renamed to `inflater_test.go`
 - Added a file-level comment
 - Renamed the `Test` function to indicate what's being tested
 - Added a comment talking about the worker's disk resizing